entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,8 @@
+export default {
+    data: {
+        title: ""
```

Comment:
API implementation is pending:
1. Use Date apis to display correct date
2. Fetch weather details from  openweather and display, https://forums.developer.huawei.com/forumPortal/en/topic/0201541977865920174
3. Integrate to fetch foot steps details from sensor and display
4. Other data can be hardcoded